src/platform_impl/android/mod.rs

```diff
@@ -143,155 +185,125 @@ impl<T: 'static> EventLoop<T> {
           }
           Event::Stop => self.running = false,
           Event::Start => self.running = true,
```

Comment:
Handlers for `Event::Start` and `EventSource::InputQueue` have been commented out. Is additional work required?

Reply:
those variants simply weren't being used